Error building from source

```
plugins/fastrouter/fastrouter.c: In function 'fr_recv_uwsgi_vars':
plugins/fastrouter/fastrouter.c:282:10: error: 'len' may be used uninitialized in this function [-Werror=maybe-uninitialized]
  ssize_t len = cr_read_exact(main_peer, pktsize+4, "fr_recv_uwsgi_vars()");
          ^
cc1: all warnings being treated as errors
make: *** [all] Error 1
INFO[0031] The command [/bin/sh -c UWSGI_PROFILE="default" UWSGI_BIN_NAME="/usr/bin/uwsgi" make] returned a non-zero code: 2
```

I am building from source from github HEAD...  in a docker image on ubuntu 14.04

it worked last time I did it, locally, which was "3 weeks ago"... now trying to build it fresh on a DigitalOcean droplet it fails with error above

not sure if it's a regression in HEAD or some difference in the platform

--- a/plugins/fastrouter/fastrouter.c
+++ b/plugins/fastrouter/fastrouter.c
@@ -241,6 +241,7 @@ static ssize_t fr_recv_uwsgi_vars(struct corerouter_peer *main_peer) {
 	struct fastrouter_session *fr = (struct fastrouter_session *) main_peer->session;
 
 	struct corerouter_peer *new_peer = NULL;
+	ssize_t len = 0;
 
 	// are we buffering ?
 	if (main_peer->is_buffering) {
@@ -267,6 +268,7 @@ static ssize_t fr_recv_uwsgi_vars(struct corerouter_peer *main_peer) {
 		// have we done ?
 		if (fr->buffered >= fr->content_length) {
 			fr->buffered = 0;
+			len = rlen;
 			goto done;
 		}
 
@@ -279,7 +281,7 @@ static ssize_t fr_recv_uwsgi_vars(struct corerouter_peer *main_peer) {
 	// increase buffer if needed
 	if (uwsgi_buffer_fix(main_peer->in, pktsize+4))
 		return -1;
-	ssize_t len = cr_read_exact(main_peer, pktsize+4, "fr_recv_uwsgi_vars()");
+	len = cr_read_exact(main_peer, pktsize+4, "fr_recv_uwsgi_vars()");
 	if (!len) return 0;
 
 	// headers received, ready to choose the instance

